keras/backend/tensorflow_backend.py

```diff
@@ -727,11 +727,41 @@ def normalize_batch_in_training(x, gamma, beta,
     return normed, mean, var
 
 
+def normalize_batch_in_testing(x, mean, var, beta, gamma,
```

Comment:
I think we should rename this to `batch_normalization` in order to follow the TF API. API consistency matters, and Keras is aligning on TF rather than Theano.